sentences/sl/_common.yaml

```
@@ -1,12 +1,49 @@
-language: sl
+language: "sl"
 responses:
   errors:
-    no_intent: TODO Sorry, I couldn't understand that
-    no_area: TODO No area named {{ area }}
-    no_domain: TODO {{ area }} does not contain a {{ domain }}
-    no_device_class: TODO {{ area }} does not contain a {{ device_class }}
-    no_entity: TODO No device or entity named {{ entity }}
-    handle_error: TODO An unexpected error occurred while handling the intent
-lists: {}
-expansion_rules: {}
-skip_words: []
+    no_intent: "Oprosti, tega nisem razumel"
+    no_area: "Območje z imenom {{ area }} ne obstaja"
+    no_domain: "{{ area }} ne obstaja v območju {{ domain }}"
+    no_device_class: "{{ area }} ne vsebuje {{ device_class }}"
+    no_entity: "Ni naprave ali entitete z imenom {{ entity }}"
+    handle_error: "Med izvajanjem je prišlo do nepričakovane napake"
+lists:
+color:
+    values:
+      - "bela"
+      - "črna"
+      - "rdeča"
+      - "oranžna"
+      - "rumena"
+      - "zelena"
+      - "modra"
+      - "vijolična"
+      - "rjava"
+  brightness:
+    range:
+      type: "odstotek"
+      from: 0
+      to: 100
+  temperature:
+    range:
+      type: "temperatura"
+      from: 0
+      to: 100
+  temperature_unit:
+    values:
+      - "celsius"
+      - in: "c"
+        out: "celsius"
+      - "fahrenheit"
+      - in: "f"
+        out: "fahrenheit"
+expansion_rules:
+  name: "{name}"
+  area: "{area}"
+  what_is: "(kaj je | kaj je to | čemu je namenjen | kateri)"
+  is_it: "(ali je)"
+  brightness: "{brightness:brightness_pct} [percent]"
+  turn_on: "( vklop | start | začni )"
+  turn_off: "( izklop | stop | končaj )"
```

We should only add lists or expansion rules when they are being used in sentences.
```suggestion
lists: {}
expansion_rules: {}
```